src/base.rs

```diff
@@ -84,8 +94,31 @@ extern "C" {
     pub static kCFAllocatorNull: CFAllocatorRef;
     pub static kCFAllocatorUseContext: CFAllocatorRef;
 
+    pub fn CFAllocatorGetTypeID() -> CFTypeID;
+
+    pub fn CFAllocatorSetDefault(allocator: CFAllocatorRef);
+    pub fn CFAllocatorGetDefault() -> CFAllocatorRef;
+
+    pub fn CFAllocatorCreate(allocator: CFAllocatorRef, context: *mut CFAllocatorContext) -> CFAllocatorContext;
+    pub fn CFAllocatorAllocate(allocator: CFAllocatorRef, size: CFIndex, hint: CFOptionFlags) -> *mut c_void;
+    pub fn CFAllocatorReallocate(allocator: CFAllocatorRef, ptr: *mut c_void, newSize: CFIndex, hint: CFOptionFlags) -> *mut c_void;
+    pub fn CFAllocatorDeallocate(allocator: CFAllocatorRef, ptr: *mut c_void);
+    pub fn CFAllocatorGetPreferredSizeForSize(allocator: CFAllocatorRef, size: CFIndex, hint: CFOptionFlags) -> CFIndex;
+    pub fn CFAllocatorGetContext(allocator: CFAllocatorRef, context: *mut CFAllocatorContext);
+
     pub fn CFGetTypeID(cf: CFTypeRef) -> CFTypeID;
+    pub fn CFCopyTypeIDDescription(cf: CFTypeID) -> CFStringRef;
 
     pub fn CFRetain(cf: CFTypeRef) -> CFTypeRef;
     pub fn CFRelease(cf: CFTypeRef);
+    pub fn CFAutorelease(arg: CFTypeRef) -> CFTypeID;
```

Comment:
This returns `CFTypeRef`:

```c
CF_EXPORT
CFTypeRef CFAutorelease(CFTypeRef CF_RELEASES_ARGUMENT arg) CF_AVAILABLE(10_9, 7_0);
```

Is `CF_RELEASES_ARGUMENT` something we need to worry about?

Reply:
Thanks for the link. Looks like it's just a hint to the static analyzer and doesn't affect functionality of the compiled code.